metagraph/src/graph/representation/hash/dbg_sshash.cpp

```diff
@@ -58,7 +58,7 @@ void DBGSSHash ::map_to_nodes_sequentially(std::string_view sequence,
     if (terminate() || sequence.size() < k_)
         return;
 
-    kmer_t uint_kmer = sshash::util::string_to_uint_kmer<kmer_t>(sequence.data(), k_ - 1);
+    kmer_t uint_kmer = sshash::util::string_to_uint_kmer<kmer_t>(std::string(sequence).c_str(), k_ - 1);
```

Comment:
since we know that the sequence size is at least `k_` at this point, and since `k_` is known by `sshash`, can we switch to using `sequence.c_str()` to avoid the copy?

Reply:
Hmm. I wrote it like this because `string_view` is not guaranteed to be null-terminated. But you're right, we don't actually rely on it in sshash, so I suppose it's fine to keep using `.data()`, as long as we're aware of potential consequences if sshash eventually starts relying on it being null-terminated (unlikely). Will update shortly.